tests/scripts/pulp_file/test_remote.sh

```diff
@@ -17,7 +17,8 @@ expect_succ pulp file remote show --remote "cli_test_file_remote"
 HREF="$(echo "$OUTPUT" | jq -r '.pulp_href')"
 test "$(echo "$OUTPUT" | jq -r '.proxy_url')" = "http://proxy.org"
 test "$(echo "$OUTPUT" | jq -r '.max_retries')" = "5"
-test "$(echo "$OUTPUT" | jq -r '.total_timeout')" = "32"
+TIMEOUT="$(echo "$OUTPUT" | jq -r '.total_timeout')"
+test 1 -eq "$(echo "${TIMEOUT}==32" | bc)"  # total_timeout is a float that can return as either 32 or 32.0
```

Comment:
I still don't understand the reason this can be returned as a float on different machines.

Reply:
Me either - but I decided "protection from Random Evironmental Weirdness" was the way to go. It's ugly - but, programming in bash just kind of *is*, so here we are.

Reply:
This may actually be jq's fault (json does not really distinguish floats from ints. It only knows "number"):
```
echo "32.0" | jq '.'
32
```